llvm/test/CodeGen/DirectX/ShaderFlags/res-may-not-alias-sm6.7.ll

```diff
@@ -2,8 +2,8 @@
 
 ; This test checks to ensure the behavior of the DXIL shader flag analysis
 ; for the flag ResMayNotAlias is correct when the DXIL Version is >= 1.7 and the
-; DXIL Validator Version < 1.8. The ResMayNotAlias flag (0x20000000) should be
-; set on all functions if there are one or more UAVs present globally in the
+; DXIL Validator Version < 1.8. The ResMayNotAlias module flag (0x20000000)
+; should be set if there are one or more UAVs present globally in the
```

Comment:
Should we add an `llc | obj2yaml` run line to this test like we have in low-precision.ll?

Reply:
There is nothing to test in the obj2yaml because ResMayNotAlias is only a module flag. It doesn't have a related shader feature flag that will show up in SFI0